src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/subjectAccessRequest/SarLicenceEventType.kt

```diff
@@ -27,4 +27,6 @@ enum class SarLicenceEventType(@JsonValue val description: String) {
   HDC_CREATED("HDC created"),
   HDC_VERSION_CREATED("HDC version created"),
   HDC_SUBMITTED("HDC submitted"),
+  HDC_VARIATION_CREATED("HDC variation created"),
```

Comment:
Some of the SAR ticket too

Reply:
Yep sorry, if I remember right the type was complaining without this